src/index.js

```diff
@@ -226,6 +226,15 @@ export default {
   get withScrollReached() {
     return require('./commons').withScrollReached;
   },
+  get BaseComponentInjectedProps() {
+    return require('./commons').BaseComponentInjectedProps;
+  },
+  get ForwardRefInjectedProps() {
+    return require('./commons').ForwardRefInjectedProps;
+  },
+  get ModifiersProps() {
+    return require('./commons').default;
+  },
```

Comment:
You need to export the typings in the ts files
`generatedTypes/index.d.ts`
`src/index.ts`

Reply:
Sorry, I don't understand what is what with all the fake files... Why they should be in `generatedTypes/index` if these are TS fils? And where all the other commons in `src/index.ts`?